frontend/src/pages/article/article.tsx

```diff
@@ -0,0 +1,11 @@
+import { type FC } from 'react';
+import { useParams } from 'react-router-dom';
```

Comment:
Import it from hooks 
```suggestion
import { useParams } from '~/libs/hooks/hooks.js';
```